src/index.ts

```diff
@@ -40,6 +52,52 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('hasOperationSupportingMultipart', function(value: CodegenOperation | CodegenOperation[]): boolean {
+			if (isCodegenOperation(value)) {
+				return containsMultipartOperation([value])
+			} else {
+				return containsMultipartOperation(value)
+			}
+		})
+
+		hbs.registerHelper('isMultipartSchema', function(value: CodegenSchema): boolean {
+			return !!value.contentMediaType?.mimeType.match('^multipart/.*')
+		})
+
+		hbs.registerHelper('isMetadataSchema', function(value: CodegenSchema): boolean {
+			return value.purpose === CodegenSchemaPurpose.METADATA
+		})
+
+		hbs.registerHelper('getFileUploadProperties', function(properties: CodegenObjectLikeSchemas[]): FileUploadProperty[] {
+			const results: FileUploadProperty[] = []
+			for (const prop in properties) {
+				const property = properties[prop]
+				if (!isCodegenSchemaUsage(property)) {
```

Comment:
Is this check necessary? The type of `property` should be `CodegenProperty` so I thought this would be redundant.

Reply:
I've removed the need for that first check by changing the input type to `CodegenProperty`.